apis/v1alpha2/policy_types.go

```diff
@@ -16,6 +16,18 @@ limitations under the License.
 
 package v1alpha2
 
+const (
+	// PolicyLabelKey is the label whose presence identifies a CRD as a policy
+	// CRD. The value of the label SHOULD be one of the following:
```

Comment:
Are we just saying SHOULD here because we don't expect to be able to conformance test it?

Reply:
Yeah good catch, `MUST` is probably correct here.

Reply:
Thanks Shane and Rob. Changed to MUST